frontend/src/styles/styles.scss

```diff
@@ -5,487 +5,512 @@
-        color: mat.get-theme-color($theme, on-primary) !important;
-    }
-
-    .primary-background-color {
-        background-color: mat.get-theme-color($theme, primary) !important;
-        color: mat.get-theme-color($theme, on-primary) !important;
-    }
-
-    .primary-color {
-        color: mat.get-theme-color($theme, primary) !important;
-    }
-
-    .secondary-background {
-        background: mat.get-theme-color($theme, secondary) !important;
-        color: mat.get-theme-color($theme, on-secondary) !important;
-    }
-
-    .primary-fixed-dim-background {
-        background: mat.get-theme-color($theme, primary-fixed-dim) !important;
-        color: mat.get-theme-color($theme, on-primary-fixed-variant) !important;
-    }
-
-    .primary-container-background {
-        background: mat.get-theme-color($theme, primary-container) !important;
-        color: mat.get-theme-color($theme, on-primary-container) !important;
-    }
-
-    .secondary-container-background {
-        background: mat.get-theme-color($theme, secondary-container) !important;
-        color: mat.get-theme-color($theme, on-secondary-container) !important;
-    }
-
-    .tertiary-container-background {
-        background: mat.get-theme-color($theme, tertiary-container) !important;
-        color: mat.get-theme-color($theme, on-tertiary-container) !important;
-    }
-
-    .secondary-background-color {
-        background-color: mat.get-theme-color($theme, secondary) !important;
-        color: mat.get-theme-color($theme, on-secondary) !important;
-    }
-
-    .secondary-color {
-        color: mat.get-theme-color($theme, secondary) !important;
-    }
-
-    .tertiary-background {
-        background: mat.get-theme-color($theme, tertiary) !important;
-        color: mat.get-theme-color($theme, on-tertiary) !important;
-    }
-
-    .tertiary-background-color {
-        background-color: mat.get-theme-color($theme, tertiary) !important;
-        color: mat.get-theme-color($theme, on-tertiary) !important;
-    }
-
-    .tertiary-color {
-        color: mat.get-theme-color($theme, tertiary) !important;
-    }
-
-    .on-secondary-background-color {
-        background-color: mat.get-theme-color($theme, on-secondary) !important;
-        color: mat.get-theme-color($theme, secondary) !important;
-    }
-
-    /// Ensures that all tables are horizontally scrollable.
-    /// All tables have to be wrapped by a div with the 
-    /// `table-responsive` class.
-    .table-responsive {
-        overflow-x: auto;
-      }
-    .table-responsive table {
-        white-space: nowrap;
-    }
+  p {
+    font: mat.get-theme-typography($theme, body-medium, font);
+  }
+
+  body {
+    ::-webkit-scrollbar {
+      width: 6px;
+    }
+
+    ::-webkit-scrollbar-track {
+      background-color: mat.get-theme-color($theme, inverse-on-surface);
+      border-radius: 10px;
+    }
+
+    ::-webkit-scrollbar-thumb {
+      background-color: mat.get-theme-color($theme, primary);

```

Comment:
This color contrasts greatly from the surrounding content, especially on light mode:

<img width="1248" alt="Screenshot 2024-08-22 at 7 30 08 PM" src="https://github.com/user-attachments/assets/a1ae4355-c657-4a31-a781-4a32fc8b7e54">

The site background color (not the background color of the pane) is `surface-container`. You can apply a slightly darker shade of this using `surface-container-high`. Consider changing the thumb color to either `surface-container` or `surface-container-high`. You can go one or two shades lighter for the track. In order from *darkest* to *lightest*:
* `surface-container-highest`